includes/class-newspack-newsletters-editor.php

```diff
@@ -45,6 +45,7 @@ public static function instance() {
 	 */
 	public function __construct() {
 		add_action( 'init', [ __CLASS__, 'register_meta' ] );
+		add_action( 'block_editor_settings_all', [ __CLASS__, 'disable_autosave' ], 10, 2 );
```

Comment:
[Nitpick but you'd technically want add_filter](https://developer.wordpress.org/reference/hooks/block_editor_settings_all/)

Reply:
Whoops, good catch! Fixed in ba2c652.

I suspect the JS-based `lockPostAutosaving()` is actually what's doing most of the heavy lifting here, but that also seems like a pretty janky way to accomplish this, so I figured some redundancy can't hurt.